src/components/DeleteAllQuestionsToggle.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-env browser */
+import React, { Fragment } from 'react'
+import { Button } from 'reactstrap'
+
+import ConfirmDeleteAllQuestionsModal from './ConfirmDeleteAllQuestionsModal'
+import QuestionNotificationsToggleExplanationModal from './QuestionNotificationsToggleExplanationModal'
+
+class DeleteAllQuestionsToggle extends React.Component {
```

Comment:
This isn't a toggle, strictly speaking. Maybe name this `DeleteAllQuestionsButton` instead?